examples/deploy-virtual-appliance-from-marketplace/main.tf

```
@@ -0,0 +1,77 @@
+provider "azurerm" {
+   subscription_id = "${var.subscription_id}"
+   client_id       = "${var.client_id}"
+   client_secret   = "${var.client_secret}"
+   tenant_id       = "${var.tenant_id}"
+}
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.resource_group}"
+  location = "${var.location}"
+}
+resource "azurerm_virtual_network" "vnet" {
+  name                = "${var.virtual_network_name}"
+  location            = "${var.location}"
```

Could we reference the rg here:
```suggestion
  location            = "${azurerm_resource_group.rg.location}"
```